### PR TITLE
Usermanagement

### PR DESCRIPTION
**Pull request by venkatkollu23**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+
+.idea
```
Comment (venkatkollu23) [inline .gitignore:5]: Why did u ignore.idea folder any reason

Comment (venkatkollu23) [reply .gitignore:5]: To ignore unwanted files to untract

Comment (venkatkollu23) [reply .gitignore:5]: To ignore unwanted files to untract

Comment (Dinesh-Adabala) [inline .gitignore:5]: why did you add .ignore file

Comment (venkatkollu23) [reply .gitignore:5]: this file is required to avoid untracked files in git

**Review by Dinesh-Adabala — APPROVED** [review]
changes look good
